adapters/meta_llama_v3_2/dataloop.json

```diff
@@ -0,0 +1,122 @@
+{
+  "name": "databricks-meta-llama-3-2",
+  "displayName": "Meta Llama 3.2",
+  "version": "0.0.13",
+  "scope": "public",
+  "description": "Llama 3.2 offers multilingual LLMs, pretrained and instruction-tuned for text-based tasks like dialogue, retrieval, and summarization.",
+  "attributes": {
+    "Provider": "Meta",
+    "Deployed By": "Databricks",
+    "License": "https://www.databricks.com/legal/open-model-license",
+    "Gen AI": "LMM",
+    "Category": "Model",
+    "NLP": "Conversational",
+    "Media Type": [
+      "Text"
```

Comment:
llama 3.2 supports images as well?

Reply:
![image](https://github.com/user-attachments/assets/b8afafa7-1df3-4c19-999c-81ed47d875d4)
 this is the llama 3.2 that are available in databricks. From meta 3.2 : 11B or 90B models for image use cases.
 https://www.llama.com/
 so the models in databricks right now are not multimodal, and also, llama 3 and 3.1 are also not multimodal:
 llama 3 : Models input text only.: https://huggingface.co/meta-llama/Meta-Llama-3-8B
 llama 3.1: Llama 3.1 (text only) https://huggingface.co/meta-llama/Llama-3.1-8B